3.resistance-signature/5.visualize-signature-features.ipynb

```diff
@@ -25,7 +25,20 @@
    "execution_count": 1,
    "id": "collectible-president",
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "ename": "ERROR",
```

Comment:
please fix. 

IIRC I don't think I used heatmap3, so you might be able to just delete the `library()` statement (I think I used `ComplexHeatmap` instead